server/pgServer.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const client = require('./pgClient.js');
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
+app.use(express.static(__dirname + '/../client/dist'));
+app.use('/*', (req,res, next) => {
+    res.header('Access-Control-Allow-Origin', '*');
+    next();
+});
+
+app.get('/api/productView/products/:id', client.getAll);
```

Comment:
This is cool! Going to need to look at express docs again to see how this works--> client.getAll is your call back?  Also though-- it seems like this is getting one product, not all right? or are you referring to getting all data for a product? Clarifying function name would make this clearer: getOne or getProduct?

Reply:
nice @claire-melbourne , I think the "getProduct" name would be much clearer for the user. Will make that change ASAP.